templates/phaser/mprocs.yaml

```
@@ -7,4 +7,4 @@ procs:
     shell: pnpm mud dev-contracts --rpc http://127.0.0.1:8545
   anvil:
     cwd: packages/contracts
-    shell: anvil --base-fee 0
+    shell: anvil --base-fee 0 -b 2
```

to make this more readable:
```suggestion
    shell: anvil --base-fee 0 --block-time 2
```